code/game/mecha/mecha.dm

```diff
@@ -108,7 +108,7 @@
 	add_airtank()
 	spark_system.set_up(2, 0, src)
 	spark_system.attach(src)
-	smoke_system.set_up(3, src)
+	smoke_system.set_up(3, FALSE, src)
```

Comment:
So this one basically just didn't work before? Can you add this to the CL if that were the case?